weatbag/tiles/e3.py

```diff
@@ -0,0 +1,45 @@
+from weatbag import words
+import random
+
+class Tile:
+    def __init__(self):
+        self.contents = {'bag of gold': 1}
+        self.blocked='w'
+
+    
+    def describe(self):
+        if self.contents['bag of gold']:
+            print("you've been teleported to a magical room. Mysterious patterns ebb and swirl on the walls.\nA bag of gold sits on the floor.\n There is a wormhole to the west.")
+        else:
+            print("you've been teleported to a magical room. Mysterious patterns ebb and swirl on the walls.\n There is a wormhole to the west.")
+            
+    def action(self, player, do):
+        if do[0] == "explode" and not self.contents['bag of gold']: 
```

Comment:
This might be a bit confusing for players - if they type 'explode self' before taking the bag of gold, they'll just see the generic "Sorry, I don't understand." response. Do you want to give them a hint that they need to take the bag first? Or just allow them to explode before taking the bag?